lib/sync-settings.coffee

```diff
@@ -28,6 +29,8 @@ module.exports =
 
   activate: ->
     # for debug
+    GitHubApi ?= require 'github'
+    PackageManager ?= require './package-manager'
```

Comment:
Does the comment make sense anyway? It's not like adding the commands is for debugging purposes only right?